### PR TITLE
update the spec to version 1.2.0

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

**Comment by surajssd** [comment]
Sample build
https://cbs.centos.org/koji/taskinfo?taskID=228576

File: kompose.spec [hunk 1]
```diff
@@ -52,7 +52,7 @@
 %global testflags -race -cover -v
 
 Name:           kompose
-Version:        1.1.0
+Version:        1.2.0
 Release:        0.1%{?dist}
```
Comment (praveenkumar) [inline kompose.spec]: @surajssd Can you make this release as 1, this is just a bump version which we usually used when a patch required. I am not sure why we have it as `0.1`?

**Review by praveenkumar — CHANGES_REQUESTED** [review]

File: kompose.spec (hunk 1 above)
Comment (surajssd) [reply kompose.spec]: done!

**Comment by surajssd** [comment]
Also I think we can use https://src.fedoraproject.org/rpms/kompose to do all the PR review process

**Comment by praveenkumar** [comment]
> Also I think we can use https://src.fedoraproject.org/rpms/kompose to do all the PR review process

@surajssd  Looks like this part recently moved to pagure, yes we should use it now on for PR. Much better, I hope here if you create PR for spec then it also run some test.

File: kompose.spec [hunk 2]
```diff
@@ -52,8 +52,8 @@
 %global testflags -race -cover -v
 
 Name:           kompose
-Version:        1.1.0
-Release:        0.1%{?dist}
+Version:        1.2.0
+Release:        1%{?dist}
```
Comment (praveenkumar) [inline kompose.spec:56]: 👍

**Review by praveenkumar — APPROVED** [review]

**Comment by abitrolly** [comment]
Should https://github.com/dustymabe/fedpkg-kompose/pull/13 close this issue?

**Comment by dustymabe** [comment]
Hey guys. There really is no reason to use this repo any longer now that dist-git is fronted by pagure. Is there any reason we are still using this repo?

**Comment by surajssd** [comment]
@dustymabe we can stop using this, it's just matter of working on the other one!

I tried create a PR to the fork of mine https://src.fedoraproject.org/fork/surajd/rpms/kompose but it fails for some reason with unauthorized error, is there a place to put my public key so that I can work with https://src.fedoraproject.org/settings .

**Pull request closed without merging**